lms/djangoapps/courseware/module_render.py

```diff
@@ -933,6 +934,9 @@ def handle_xblock_callback(request, course_id, usage_id, handler, suffix=None):
 
     try:
         course_key = CourseKey.from_string(course_id)
+        if isinstance(course_key, CCXLocator):
+            course_key = course_key.to_course_locator()
+
```

Comment:
1- do you need to put this code in the `try/except`? 
2- you do not need the empty line